test/phpunit.xml.dist

```diff
@@ -2,9 +2,9 @@
 <phpunit bootstrap="bootstrap.php">
   <testsuites>
     <testsuite name="General">
-      <file>./smtpapi.php</file>
-      <file>./smtpapi/Header.php</file>
-      <file>./RepoFilesExists.php</file>
+      <file>./Smtpapi.php</file>
+      <file>./Smtpapi/Header.php</file>
+      <!--<file>./RepoFilesExists.php</file>-->
```

Comment:
It seems that this file is not used because this file is under the comment.

Is it necessary to keep this file with the comment in `phpunit.xml.dist` file? I think it should be removed from this file :-).

Reply:
I can remove it, I left it in hoping someone might be working to fix that test 😁 I think it was accidentally merged into master before it was ready.